pkgs/development/libraries/gdal/default.nix

```
@@ -252,37 +264,36 @@ stdenv.mkDerivation (finalAttrs: {
     "gdrivers/gdalhttp.py"
     "gdrivers/wms.py"
   ];
-  disabledTests = [
-    # tests that attempt to make network requests
-    "test_jp2openjpeg_45"
-    # tests that require the full proj dataset which we don't package yet
-    # https://github.com/OSGeo/gdal/issues/5523
-    "test_transformer_dem_overrride_srs"
-    "test_osr_ct_options_area_of_interest"
-    # ZIP does not support timestamps before 1980
-    "test_sentinel2_zipped"
-    # tries to call unwrapped executable
-    "test_SetPROJAuxDbPaths"
-    # fixed and renamed in 3.8.0RC1
-    # https://github.com/OSGeo/gdal/commit/c8b471ca1e6318866ff668d2b57bb6f076e3ae29
-    "test_visoss_6"
-    # failing with PROJ 9.3.1
-    # https://github.com/OSGeo/gdal/issues/8908
-    "test_osr_esri_28"
-    # failing for unknown reason
-    # https://github.com/OSGeo/gdal/pull/10806#issuecomment-2362054085
-    "test_ogr_gmlas_billion_laugh"
-  ] ++ lib.optionals (!stdenv.hostPlatform.isx86_64) [
-    # likely precision-related expecting x87 behaviour
-    "test_jp2openjpeg_22"
-  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    # flaky on macos
-    "test_rda_download_queue"
-  ] ++ lib.optionals (lib.versionOlder proj.version "8") [
-    "test_ogr_parquet_write_crs_without_id_in_datum_ensemble_members"
-  ] ++ lib.optionals (!usePoppler) [
-    "test_pdf_jpx_compression"
-  ];
+  disabledTests =
+    [
+      # tests that attempt to make network requests
+      "test_jp2openjpeg_45"
+      # tests that require the full proj dataset which we don't package yet
+      # https://github.com/OSGeo/gdal/issues/5523
+      "test_transformer_dem_overrride_srs"
+      "test_osr_ct_options_area_of_interest"
+      # ZIP does not support timestamps before 1980
+      "test_sentinel2_zipped"
+      # tries to call unwrapped executable
+      "test_SetPROJAuxDbPaths"
+      # failing for unknown reason
+      # https://github.com/OSGeo/gdal/pull/10806#issuecomment-2362054085
+      "test_ogr_gmlas_billion_laugh"
+    ]
```

This would be an opportunity to disable a few flaky tests that were collected in https://github.com/NixOS/nixpkgs/pull/327323.

```suggestion
      # Flaky on hydra, collected in https://github.com/NixOS/nixpkgs/pull/327323.
      "test_ogr_gmlas_huge_processing_time"
      "test_ogr_gpkg_background_rtree_build"
      "test_vsiaz_fake_write"
      "test_vsioss_6"
    ]
```

---

Looking at how they mark them upstream, wondering if we could set `env.CI=true` or something to make sure we skip anything they flag as flaky to lower maintenance burden.

---

Probably a good idea, but we should check if our list contains anything that is not marked as flaky upstream. We could also ask if they'd consider disabling these tests by default instead.